discovery/roles/postscripts/ubuntu/templates/omnia_intelgaudi.j2

```diff
@@ -70,7 +70,7 @@ if [[ $gaudi_check_processing_xlr8r == *"Habana Labs Ltd"* ]]; then
     echo "deb [trusted=yes] http://{{ admin_nic_ip }}:80/install{{ repo_store_path }}/cluster/apt/intelgaudi/{{ intelgaudi_version }} ./" >> /etc/apt/sources.list.d/intelgaudi.list
 
     sudo apt-get update
-    sudo apt install dkms libelf-dev -y
+    sudo apt install libopenmpi3 dkms libelf-dev -y
```

Comment:
Why this package is required? Any failure in installation? Do we need this changes in accelerator flow as well?